scripts/update-rdpSettings

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+
+# This scripts reads include/freerdp/settings.h and scans the
+# rdp_settings structure for all the fields until the /END of ABI
+# stable zone/ comment.  It then generates a set of #define one per
+# non-padding field, valued as the index of the field, taking into
+# account padding fields.  These field #defines are updated in that
+# same file.
+#
+# Usage:
+#
+#    cd $freerdp_source_directory
+#    scripts/update-rdpSettings
+
+file="include/freerdp/settings.h"
```

Comment:
might be a good idea to make this path relative to the script path `SCRIPT_PATH=$(dirname "${BASH_SOURCE[0]}")
 SCRIPT_PATH=$(realpath "$SCRIPT_PATH")` or something similar should do the trick.